api-report/container-definitions.api.md

```diff
@@ -75,6 +75,13 @@ export interface IAudience extends EventEmitter {
     on(event: "addMember" | "removeMember", listener: (clientId: string, client: IClient) => void): this;
 }
 
+// @public (undocumented)
+export interface IAudienceWriter extends IAudience {
```

Comment:
Maybe IAudienceOwner? Not sure.
If it's assumed that there is always single writer, then "owner" is more appropriate IMHO in naming like that.
Though consumer/producer (as well as reader/writer) could also be used in scenarios with single writer.

Reply:
Naming is not easy..